apps/console/src/features/identity-providers/components/forms/authenticators/saml-authenticator-settings-form.tsx

```diff
@@ -0,0 +1,829 @@
+
+                <GridItem width={ 13 }>
+                    <FormSection heading="Advanced">
+                        <Grid>
+
+                            <GridItem width={ 16 }>
+                                <FinalFormField
+                                    toggle
+                                    required={ false }
+                                    name="IncludeProtocolBinding"
+                                    aria-label={ "Include protocol binding in the request" }
+                                    data-testid={ `${ testId }-IncludeProtocolBinding-field` }
+                                    component={ SCheckboxAdapter }
+                                    label={ (
+                                        <RichLabel htmlFor="IncludeProtocolBinding">
+                                            Include protocol binding in the request
+                                        </RichLabel>
+                                    ) }
+                                />
+                                <Hint icon="info circle">
+                                    Specifies whether the transport mechanism should be included in the SAML
+                                    request assertion.
+                                </Hint>
+                            </GridItem>
+
+                            <GridItem width={ 16 }>
+                                <FinalFormField
+                                    required={ false }
+                                    name="IsUserIdInClaims"
+                                    aria-label={ "Use Name ID as the user identifier." }
+                                    data-testid={ `${ testId }-IsUserIdInClaims-field` }
+                                    toggle
+                                    component={ SCheckboxAdapter }
+                                    label={ (
+                                        <RichLabel htmlFor="IsUserIdInClaims">
+                                            { values.IsUserIdInClaims
+                                                ? (<span>User identifier found among <Code>claims</Code></span>)
+                                                : (<span>Use <Code>NameID</Code> as the user identifier</span>)
+                                            }
+                                        </RichLabel>
+                                    ) }
+                                />
+                                <Hint icon="info circle">
+                                    If you need to specify an attribute from the SAML assertion as the User Identifier,
+                                    you can uncheck this option and configure the <Code>subject</Code> from
+                                    the Attributes section.
+                                </Hint>
+                            </GridItem>
+
+                            <GridItem width={ 16 }>
+                                <RichLabel htmlFor="commonAuthQueryParams">
```

Comment:
Isn't it possible to use the semantic label here?

Reply:
[SemanticUI Label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) != [HTML5 label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

Reply:
Since we don't have a common one I will move this component to `react-components` module.